ccmlib/common.py

```diff
@@ -908,14 +908,9 @@ def get_supported_jdk_versions(cassandra_version, install_dir, for_build, env):
             run_versions = [11, 17]
             info("Cassandra 5.0+ detected, using Java 11 and 17")
         elif cassandra_version and cassandra_version >= LooseVersion('4.0'):
-            if 'CASSANDRA_USE_JDK11' not in env or env['CASSANDRA_USE_JDK11'] != 'true':
-                build_versions = [8, 11]
-                run_versions = [8, 11]
-                info("Cassandra 4.0+ detected, using Java 8 and 11")
-            else:
-                build_versions = [11]
-                run_versions = [11]
-                info("Cassandra 4.0+ detected, using Java 11")
+            build_versions = [8, 11]
+            run_versions = [8, 11]
+            info("Cassandra 4.0+ detected, using Java 8 and 11")
```

Comment:
oh, when we build ccm it sets the `-Duse.jdk11=true` flag

Reply:
so, will this still break the following situation…?

The env is path+java_home 8, java11_home is set, CASSANDRA_USE_JDK11 is true, ccm source 4.0

ccm goes to build cassandra-4.0 using jdk8.  it fails because CASSANDRA_USE_JDK11 is true.

Reply:
the existing contract/expectation here was that `CASSANDRA_USE_JDK11=true` mean switch to use jdk11